experimental/PropertyDDS/examples/partial-checkout/package.json

```diff
@@ -122,7 +122,7 @@
 		},
 		"buildDependencies": {
 			"merge": {
-				"tsc": {
+				"build:esnext": {
 					"@fluid-experimental/property-inspector-table": [
 						"build:webpack"
 					]
```

Comment:
Note:  This demo app consumes the post-webpacked output of the 'property-inspector-table', which I think may be an anti-pattern.

I didn't look at this too closely, since once #20052 lands, we should be free to delete this demo.

@ChumpChief - FYI.